content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -430,8 +430,8 @@ Please note that:
 
 A ServiceAccount for `kube-proxy` is created in the `kube-system` namespace; then kube-proxy is deployed as a DaemonSet:
 
-- The credentials (`ca.crt` and `token`) to the master come from the ServiceAccount
-- The location of the master comes from a ConfigMap
+- The credentials (`ca.crt` and `token`) to the control plane come from the ServiceAccount
+- The location (URL) of the API server comes from a ConfigMap
```

Comment:
Possible edit:
The endpoint for the API server is stored in a ConfigMap OR
The API server URL [endpoint] is saved in [retrieved from] a ConfigMap

Reply:
Which is right?

Reply:
Does "comes from" imply the value is taken from a value in the ConfigMap?

Reply:
I'd have to look at the source code or something to check - I was planning to leave the original text pretty much as-is.

Reply:
OK. Leave further edits to another PR.